components/automate-ui/src/app/components/sidebar/sidebar.component.spec.ts

```diff
@@ -34,53 +35,55 @@ describe('SidebarComponent', () => {
       ]
     }).compileComponents();
     store = TestBed.inject(Store);
-    layoutFacade = TestBed.inject(LayoutFacadeService);
```

Comment:
This instantiation was occurring too early -- moved to be after the feature flag is set.